package.json

```diff
@@ -64,6 +65,7 @@
     "jest-html-reporter": "^3.10.2",
     "jest-mock-extended": "^3.0.7",
     "mochawesome": "^7.1.3",
+    "nhs-notify-web-template-management-ui-tests": "file:tests/test-team",
```

Comment:
Not sure why this keeps popping up. I don't think the web-app needs a reference to this tests. Can you remove?